content/integrate/redis-data-integration/installation/install-k8s.md

```diff
@@ -61,6 +61,10 @@ information. *This requires Redis Enterprise v6.4 or greater*.
 - Use the Redis console to create a database with 250MB RAM with one primary and one replica.
 - If you are deploying RDI for a production environment then secure this database with a password
   and [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security).
+- Set the database's
+  [eviction policy]({{< relref "/operate/rs/databases/memory-performance/eviction-policy" >}}) to `noeviction` and set
+  [data persistence]({{< relref "/operate/rs/databases/configure/database-persistence" >}})
+  to AOF - fsync every 1 sec.
```

Comment:
should be also surrounded with backticks, no?

Reply:
@spicy-sauce You can set these from the Cluster manager UI as well as the CLI. The linked pages give more explanation, so I was trying to follow the existing formatting (for example, [this table](https://redis.io/docs/latest/operate/rs/databases/configure/database-persistence/#data-persistence-options)). The eviction policies are listed using the key name, which is why I used backticks for that.

However, if you think backticks for all of these would be clearer then I'll be happy to add them.